marklogic-langchain4j/src/main/java/com/marklogic/langchain4j/splitter/JsonChunkDocumentProducer.java

```diff
@@ -80,4 +84,24 @@ protected DocumentWriteOperation makeChunkDocument() {
     private String determineChunksArrayName(ObjectNode doc) {
         return doc.has(DEFAULT_CHUNKS_ARRAY_NAME) ? "splitter-chunks" : DEFAULT_CHUNKS_ARRAY_NAME;
     }
+
+    private ArrayNode generateConceptsForChunk(String text) {
+        // TODO - How should we handle these exceptions?
```

Comment:
You can remove this, Sonar will complain about it otherwise, we'll revisit everything after we get feedback from S4 team.